src/components/Dashboard/Searchbar.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { KeyboardEvent, useState } from 'react';
+import styles from '@/components/Dashboard/Dashboard.module.scss';
+
+function Searchbar({ label, handleSearch }: searchProps) {
```

Comment:
why send function as a prop? cant we directly import it?

Reply:
For better testing

Reply:
I have followed tdd here so tests were written before hand: 
- https://github.com/Real-Dev-Squad/website-status/pull/547

Reply:
done, imported from util